prometheus_hardware_exporter/__main__.py

```diff
@@ -77,17 +104,40 @@ def parse_command_line() -> argparse.Namespace:
     return args
 
 
-def main(port: int, level: str, enable_collectors: List[str], daemon: bool = False) -> None:
+# pylint: disable=R0913
+def main(
+    port: int,
+    level: str,
+    enable_collectors: List[str],
+    redfish_host: str,
```

Comment:
redfish_xxx is only used by redfish. 
Is it possible that we can encapsulated the rest of parameters as a config dictionary and pass into collector?